### PR TITLE
Warped logic bio

### PR DESCRIPTION
**Pull request by WarpedLogic**
review and add my bio

### CODE REVIEW

File: docs/WarpedLogic.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Szymon Jaworski
+
+Software Engineer
+
+Milwaukee, WI
```
Comment (WarpedLogic) [inline docs/WarpedLogic.md]: ...

**Comment by WarpedLogic** [comment]
testing pull request @EBHero

File: docs/WarpedLogic.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Szymon Jaworski
```
Comment (abhi-shukla) [inline docs/WarpedLogic.md:1]: Some file header might be good

File: docs/WarpedLogic.md (hunk 1 above)
Comment (abhi-shukla) [inline docs/WarpedLogic.md:5]: :+1:  Good work. Here is your reward :sweet_potato:

**Comment by EBHero** [comment]
Sounds legit.

**Pull request merged**